src/components/backend-ai-credential-view.ts

```diff
@@ -1018,6 +1028,20 @@ export default class BackendAICredentialView extends BackendAIPage {
           ></mwc-button>
         </div>
       </backend-ai-dialog>
+      ${this.openUserSettingModal
+        ? html`
+            <backend-ai-react-user-setting-dialog
+              value="${JSON.stringify({
+                open: this.openUserSettingModal,
+              })}"
+              @ok="${() => {
+                this.openUserSettingModal = false;
+                this.activeUserList.refresh();
+              }}"
+              @cancel="${() => (this.openUserSettingModal = false)}"
```

Comment:
> [!NOTE]  
> This will be moved to react component in #2757

Reply:
Thanks for the mention. I've add a description of it in the body of the PR.